### PR TITLE
Switch to SurfaceTopography v1.0

### PR DESCRIPTION
**Pull request by pastewka**
This PR implements API changes from latest SurfaceTopography

### CODE REVIEW

File: topobank/manager/views.py [hunk 1]
```diff
@@ -312,9 +312,7 @@ def clean_channel_name(s):
             # Set data source choices based on file contents
             #
             kwargs['data_source_choices'] = [(k, clean_channel_name(channel_info.name)) for k, channel_info in
-                                             enumerate(channel_infos)
```
Comment (pastewka) [inline topobank/manager/views.py:315]: Hi @mcrot - please have a look at this change. I did not undertstand what the condition was good for. Unit information in the `info` dictionary is now gone.

**Review by mcrot — DISMISSED** [review]
Looks good!

**Comment by mcrot** [comment]
I'm not sure which PR should be merged as first. Probably this one?

**Comment by pastewka** [comment]
This is because I think upgrading the dependencies may lead to unforseen issues.

**Review by mcrot — APPROVED** [review]

**Pull request merged**